pulsar-broker/src/test/java/org/apache/pulsar/broker/BrokerTestUtil.java

```diff
@@ -29,4 +31,8 @@ public static String newUniqueName(String prefix) {
         return prefix + "-" + UUID.randomUUID();
     }
 
+    public static String randomSuffixString(String content, int numSuffix) {
+        return content + "-" + RandomStringUtils.randomAlphabetic(numSuffix).toLowerCase();
```

Comment:
why newUniqueName is not a good fit for you ?

Reply:
The method `newUniqueName ` could work well, but the UUID suffix is hard to read and check, maybe a random alphabetic string is enough. WDYT?

Reply:
we can enhance newUniqueName in a follow up patch if you prefer.
at the moment it is better to have only one single way of generating random topic names